### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by pmkovar**

### CODE REVIEW

File: source/rdo/release-cadence.html.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The OpenStack package maintainer team keeps only the latest OpenStack developmen
 Due to the slower release cadence of RHEL and therefore CentOS, only the most recent release plus the previous one are supported, in keeping up with upstream policy. Older versions of the operating system such as CentOS 6 do not receive later releases.
 
 |----------------------|----------|-----------|
-| RHEL 7 and CentOS 7  | Mitaka   | supported |
```
Comment (amoralej) [inline source/rdo/release-cadence.html.md]: Mitaka is still supported.

Comment (pmkovar) [inline source/rdo/release-cadence.html.md]: @amoralej Thanks for pointing that out. When's the Mitaka EOL then?

Comment (amoralej) [inline source/rdo/release-cadence.html.md]: According to https://releases.openstack.org/ 04-2017

Comment (pmkovar) [inline source/rdo/release-cadence.html.md]: Fixed.

**Pull request merged**